pkg/image/registry/imagestream/strategy_test.go

```diff
@@ -312,6 +312,7 @@ func TestTagVerifier(t *testing.T) {
 			}
 			expectedSar := &authorizationapi.SubjectAccessReview{
 				Action: authorizationapi.Action{
+					Group:        "image.openshift.io",
```

Comment:
depends on what you have here: https://github.com/mfojtik/origin/blob/2259dd5022d7ff7af22d53d4d3214e935c19bdf5/pkg/image/registry/imagestream/strategy.go#L447

I changed it to api.LegacyGroupName which should preserve the legacy behavior, but I think we should really be converting ourself slowly towards API groups.